mlir/include/mlir/Dialect/AMDGPU/IR/AMDGPU.td

```diff
@@ -433,6 +433,46 @@ def AMDGPU_LDSBarrierOp : AMDGPU_Op<"lds_barrier"> {
   let assemblyFormat = "attr-dict";
 }
 
+def AMDGPU_SchedBarrierOpOpt : I32BitEnumAttr<"sched_barrier_opt_enum",
+    "The possible options for scheduling barriers",
+    [
+      I32BitEnumAttrCaseBit<"allow_none", 0>,
```

Comment:
Note that `0` here means the `0`-th bit is set, not meaning all bits are unset (which is `I32BitEnumAttrCaseNone`. Just wanted to double check--do we actually have the `0`-th bit set to mean allowing nothing?

Reply:
I32BitEnumAttrCaseNone is the right choice here. thanks! fixed now.